.github/actions/health_check_action.yml

```
@@ -0,0 +1,16 @@
+    description: 'Health check URL'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - shell: bash
+      run: |
+        for i in {1..20}; do
+          curl --fail -v "${{ inputs.url }}" && exit 0
+          sleep 5
+        done
+        exit 1
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)